### ISSUE TITLE
FE4: "Retain Healers" can miss the point of retaining healers

### ISSUE THREAD

**Issue by Pushwall** [lushen124/Universal-FE-Randomizer#119]
Their starting healing staves can randomize into something that isn't a healing staff, even if Retain Healers is on. I generated a seed with this option and Aideen randomized into a Priestess with a Light tome in place of Mend, and Claude randomized into a Sage with a Fire tome in place of Fortify. None of the drops or shops in the first generation gave any healing staves either; the first one I got access to was in Lana/Muirne's starting inventory, well out of the reach of Aideen/Claude, which makes it pretty hard for them to be healers.

**Comment by lushen124** [comment]
Weird... there's logic explicitly to make sure that any character that can use staves will have a healing staff in their inventory (when possible). What class was Edain or Claude? Do you have a changelog?

**Comment by Pushwall** [comment]
No changelog for the original, sadly, but Aideen was a Priestess/Shaman/whatever you call Deirdre's vanilla class, and Claude was a Sage.

However, I've got you a changelog of a different seed that also claims no healing staves exist in the first generation (unless you count Claude's Valkyrie), where Aideen is a Troubadour and Claude is a High Priest. I've played up to Aideen's recruitment to confirm that she isn't sneakily getting the mysterious Heal staff in item slot 0x12 which claims to not be available anywhere, and don't feel like playing another potential "no healing staves in first gen" seed further to confirm that it really does have no healing staves in the first generation.

[fe4rando3.zip](https://github.com/lushen124/Universal-FE-Randomizer/files/3100879/fe4rando3.zip)
